src/pages/Register/Register.tsx

```diff
@@ -195,6 +193,23 @@ const Register = ({ setUser }: RegisterProps) => {
                 handleInput={handlePasswordInput}
               />
             </div>
+            <div>
+              {firebaseError === "emailError" && (
+                <>
```

Comment:
You do not need a react fragment `<>` as you are displaying one p tag, not multiples.